### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kerautret**
add description about the main organisation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
 # DGtalTools-contrib
-Various tools from the community built using DGtal
+Various tools from the community built using DGtal.
+
+
```
Comment (dcoeurjo) [inline README.md]: maybe add a sentence to distinguish DGtaltools-contrib from DGtalTools ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+   - basicEditMesh: to apply basic mesh edition (scale change, mesh face contraction, face filtering).
+   - compareErrorMesh: compute hausdorf distance between two meshes.
+
+
+
+Visualisation
+-------------
+
+This section, can contains all tools related to visualisation. 
+Actually it contains this tool:
```
Comment (dcoeurjo) [inline README.md]: maybe you can remove this sentence

File: README.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+
+ - [Visualisation](#visualisation)
+
+
+
+Geometry2d
+----------
+
+This section can contains various utilities related to 2d geometry (2d estimators, 2d contours tools, ...)  
+Actually it contains this tool:
```
Comment (dcoeurjo) [inline README.md]: remove this sentence as well?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+ - [Geometry3d](#geometry3d)
+
+ - [Visualisation](#visualisation)
+
+
+
+Geometry2d
+----------
+
+This section can contains various utilities related to 2d geometry (2d estimators, 2d contours tools, ...)
```
Comment (dcoeurjo) [inline README.md]: 2D contour tools

File: README.md (hunk 1 above)
Comment (kerautret) [inline README.md]: yes

**Comment by kerautret** [comment]
thanks for the remarks, done

**Comment by kerautret** [comment]
@dcoeurjo can I merge ?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,40 @@
...
+   - meaningFullThickness: to display the meaningful thickness of digital contour.
+
+
+Geometry3d
+----------
+
+As the previous section but in 3d, it contains actually these tools:
+
+   - basicEditMesh: to apply basic mesh edition (scale change, mesh face contraction, face filtering).
+   - compareErrorMesh: compute hausdorf distance between two meshes.
```
Comment (dcoeurjo) [inline README.md]: hausdorff ;)

Comment (dcoeurjo) [inline README.md]: does it really compute mesh hausdorff distance or it is hausdorff distance of point sets (vertices) ?

File: README.md [hunk 6]
```diff
@@ -1,3 +1,40 @@
...
+ - [Visualisation](#visualisation)
+
+
+
+Geometry2d
+----------
+
+This section can contains various utilities related to 2d geometry (2d estimators, 2d contour tools, ...):
+
+   - meaningFullThickness: to display the meaningful thickness of digital contour.
```
Comment (dcoeurjo) [inline README.md]: maybe you can add citation ?

File: README.md [hunk 7]
```diff
@@ -1,3 +1,40 @@
 # DGtalTools-contrib
-Various tools from the community built using DGtal
+
+Various tools from the community built using DGtal. Contrary the to DGtalTools project, these tools are not necessary final but can be considered as development tools or prototypes used to share recent research in progress.
```
Comment (dcoeurjo) [inline README.md]: to the ?

File: README.md (hunk 5 above)
Comment (kerautret) [inline README.md]: In fact it computes the hausdorff /2 from shape A to B. (not from the vertices).
By the way the computation was not full correct in all cases.I change it with a better more complete version which computes a better approximation in various cases.
And I also rename it with a better name (computeMeshDistances)

File: README.md (hunk 6 above)
Comment (kerautret) [inline README.md]: yes agree thanks ;)

File: README.md (hunk 7 above)
Comment (kerautret) [inline README.md]: yes ;)

**Comment by kerautret** [comment]
ping @dcoeurjo ready with some improvements of the computeMeshDistances which now is fine and can  be tested with some adds of samples. To test:
./build/geometry3d/computeMeshDistances -i Samples/fandisk.off -c Samples/box.off -o distances.dat -m res.off --maxScaleDistance 0.7 -n 
you will obtain;
capture d ecran 2015-12-17 a 10 28 43
(note is need the PR #1091 of DGtal

**Comment by kerautret** [comment]
@dcoeurjo can I merge ? (it is just to simplify other update of readme for instance ;) )

**Comment by dcoeurjo** [comment]
Yep. thanks for the PR

**Pull request merged**